src/main/java/org/jabref/gui/commonfxcontrols/FieldFormatterCleanupsPanelViewModel.java

```diff
@@ -0,0 +1,78 @@
+package org.jabref.gui.commonfxcontrols;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.scene.control.SelectionModel;
+
+import org.jabref.Globals;
+import org.jabref.gui.util.NoSelectionModel;
+import org.jabref.logic.cleanup.Cleanups;
+import org.jabref.model.cleanup.FieldFormatterCleanup;
+import org.jabref.model.cleanup.Formatter;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.FieldFactory;
+
+public class FieldFormatterCleanupsPanelViewModel {
+
+    private final BooleanProperty cleanupsDisableProperty = new SimpleBooleanProperty();
+    private final ListProperty<FieldFormatterCleanup> cleanupsListProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+    private final ObjectProperty<SelectionModel<FieldFormatterCleanup>> selectedCleanupProperty = new SimpleObjectProperty<>(new NoSelectionModel<>());
+    private final ListProperty<Field> availableFieldsProperty = new SimpleListProperty<>(FXCollections.observableArrayList(FieldFactory.getCommonFields()));
+    private final ObjectProperty<Field> selectedFieldProperty = new SimpleObjectProperty<>();
+    private final ListProperty<Formatter> availableFormattersProperty = new SimpleListProperty<>(FXCollections.observableArrayList(Cleanups.getBuiltInFormatters()));
+    private final ObjectProperty<Formatter> selectedFormatterProperty = new SimpleObjectProperty<>();
+
+    public FieldFormatterCleanupsPanelViewModel() {
+    }
+
+    public void resetToRecommended() {
+        Globals.stateManager.getActiveDatabase().ifPresent(databaseContext -> {
```

Comment:
Pass state manager as Parameter, it can be injected into the view

Reply:
Overlooked that, thanks.